molten-platform/pom.xml

```diff
@@ -56,7 +57,8 @@
     <testng.version>7.3.0</testng.version>
     <assertj.version>3.18.1</assertj.version>
     <hamcrest.version>2.0.0.0</hamcrest.version>
-    <mockito.version>3.3.3</mockito.version><!-- 3.4.0 has a breaking change -->
```

Comment:
As this is a breaking change, I'd consider bumping molten version. At least the minor one.

Reply:
Bumped in e82d08dd.